### DISCUSSION TITLE
Node and Windows 11 ISSUE

### DISCUSSION THREAD

**Discussion by HavanaMi** [community/community#139801] [GitHub Education] [answered]

**Comment by davevad93** [comment, marked as answer]
Hi @HavanaMi ,

Install `Node.js` using the [prebuilt installer](https://nodejs.org/en/download/prebuilt-installer/current) choose your preferred version, I use `20.17.0` because it's `LTS` but that's up to you, during the installation, leave all the default options checked (don't check `Automatically install the necessary tools`), then you have to add the path to the environment variables, just follow these steps:

1. Go to `Advanced system settings`.
2. Click on `Environment Variables`.
3. Under `System variables` click `New`.
4. Enter `nodejs` as the variable name and the installation path of node (default is `C:\Program Files\nodejs`) as the variable value and save.

Now check that everything has been installed correctly by running `node -v` and `npm -v`.

Here's also a [video tutorial on Youtube](https://www.youtube.com/watch?v=1Rmb6bkGpG0) which basically covers the same steps if you need it.

**Comment by maheerCodes** [comment]
Thanks for sharing this solution, @davevad93.

@HavanaMi, could you let us know which specific error you're encountering?

A few details that may help narrow it down:

* Which Node.js version are you trying to install?
* What output do you get from `node -v` and `npm -v` in Command Prompt?
* Do you receive a "node is not recognized" error, or is the issue related to GitHub Education/Copilot coursework requirements?
* Have you already restarted your terminal or system after installation?

If the steps above resolved the issue for you, please consider marking the answer as accepted so other students facing the same problem can find the solution more easily.

Otherwise, feel free to share the exact error message and we'll try to help further.
